### ISSUE TITLE
Braille display not recognised after PC wakes from sleep

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#15942]
### Steps to reproduce:

1. User has a Focus Blue braille display.  Connected via USB and working with NVDA.
2. PC goes to sleep mode.
3. User wakes PC from sleep

### Actual behavior:

At step 3, the Braille display no longer works, until user restarts NVDA.

Comments from user: "If I start the computer from scratch, the Focus is recognized and starts up. But when the computer goes asleep, the Focus does so too (that is normal), but it does not wake up again, when the computer does. Or to be more precisely, the Focus can be brought back to life, but not in connection with NVDA. Switching the display on and off or un and replugging the display does not work. Only a restart of NVDA brings the thing back to life"

### Expected behavior:

Braille display should work as normal.

I note there is an issue for this with a Brailliant display: https://github.com/nvaccess/nvda/issues/7736
There is also an issue for this with HID displays: https://github.com/nvaccess/nvda/issues/13485

On the HID issue, @Adriani90 suggested the Brailliant issue may be related.

#### NVDA version:
NVDA 2023.3 Installed

#### Windows version:

Windows 11 Pro

#### Does the issue still occur after restarting your computer?

Yes

**Comment by seanbudd** [comment]
Do you know if this is a recent regression - i.e. can this be reproduced in 2023.1?

**Comment by seanbudd** [comment]
To be able to process this issue we need a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences and hwIO logging enabled in advanced preferences.

**Comment by Qchristensen** [comment]
I've requested a log from the user and will update as I get more information (Kim via email).

User did advise that this did not happen in older versions of NVDA (but wasn't any more specific than that).

**Comment by Qchristensen** [comment]
Follow up from the user advising that they no longer have this display so can't reproduce the issue.  Since no-one who has commented on this so far has experienced it, I'm going to close the issue.  If anyone else encounters this, and can supply a log, please feel free to reopen or create a new issue.
